src/System.Reflection.TypeExtensions/src/System.Reflection.TypeExtensions.csproj

```diff
@@ -29,7 +28,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Release|AnyCPU'" />
-  <ItemGroup Condition="'$(GeneratePlatformNotSupportedAssembly)' != 'true'">
+  <ItemGroup Condition="'$(GeneratePlatformNotSupportedAssemblyMessage)' == ''">
```

Comment:
As above please change to ` Condition="'$(TargetGroup)' != 'netstandard1.3' and '$(TargetGroup)' != 'netstandard1.5'"`